src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
```

Comment:
The route for '/home' is currently redirecting to '/'. If '/home' and '/' are meant to render the same component, consider removing the '/home' route to avoid unnecessary redirection.